backend/src/main/java/de/ffmjava/capstone/backend/user/CustomApiErrorHandler.java

```diff
@@ -35,4 +37,20 @@ public CustomApiErrorHandler(Errors errors) {
         this.errors = errors;
     }
 
+    @Nullable
+    public static ResponseEntity<Object> handlePossibleErrors(Errors errors) {
+        if (errors.hasErrors()) {
+            FieldError fieldError;
+            String errorMessage = null;
+            if (errors.getFieldError() != null) {
+                fieldError = errors.getFieldError();
+                if (fieldError != null) {
```

Comment:
unnecessary check for fieldError:  errors.getFieldError() already not null.

Reply:
SonarLint forced me to do that